packages/core/lib/test.js

```diff
@@ -225,14 +251,21 @@ const Test = {
     accounts,
     testResolver,
     runner,
-    compilations
+    compilations,
+    bugger //for stacktracing
   }) {
     global.interfaceAdapter = interfaceAdapter;
     global.web3 = web3;
     global.assert = chai.assert;
     global.expect = chai.expect;
     global.artifacts = {
-      require: import_path => testResolver.require(import_path)
+      require: import_path => {
```

Comment:
BTW for next time, feel free to change the local variables to camelcase while you're at it in situations like this ;)

Reply:
Heh, OK. :)